screens/modals/ResponderHelpRequestModal.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from "react";
+import {
+  StyleSheet,
+  Button,
+  AppState
+} from "react-native";
+import { Actions } from "react-native-router-flux";
+import { Container, Content, Header, View } from "../../components/layout";
+import { Text } from "../../components/typography";
+
+const ResponderHelpRequestModal = (props) => {
+
+  if (AppState.currentState.match(/inactive|background/)) return null;
```

Comment:
When I have the app backgrounded and I open the notification, I get a blank screen.
Commenting out this line fixes it.
I've never run into the problem of the modal opening twice.
![86294630_847179999075419_4779318785263796224_n](https://user-images.githubusercontent.com/42985875/74576497-ddbe6d80-4f3f-11ea-92f3-8a773d2a5a54.png)

Reply:
so iOS there is no issues of it popping out twice?

Reply:
not that i know of

Reply:
Hmm good to know... I will set for Android only